tests/sentinel/tests/00-base.tcl

```diff
@@ -208,3 +208,13 @@ test "SENTINEL RESET can resets the primary" {
     assert_equal 0 $res2
     assert_equal 0 $res3
 }
+
+test "SENTINEL IS-PRIMARY-DOWN-BY-ADDR checks if the primary is down" {
```

Comment:
btw, in this base.tcl, why the primary is down?

Reply:
This is only for testing this IS-PRIMARY-DOWN-BY-ADDR command.